config/kraft/broker.properties

```diff
@@ -185,24 +185,20 @@ s3.stream.set.object.compaction.force.split.minutes=120
 # The maximum stream set objects allowed to be compacted in one execution, default 500
 s3.stream.set.object.compaction.max.num=500
 
-# The baseline network bandwidth of the broker, default 100MB. This is used to throttle the network usage during compaction
+# The baseline network bandwidth of the broker in bytes/s, default 100MB/s. This is used to throttle the network usage during compaction
+# and catch up read
 s3.network.baseline.bandwidth=104857600
 
-# Uncomment the following line to enable the s3 metrics log reporter
-# metric.reporters=org.apache.kafka.server.metrics.s3stream.KafkaS3MetricsLoggerReporter
-# The interval at which the s3 metrics are reported
-# s3.metrics.logger.interval.ms=60000
-
 ############################# Settings for Auto Balancer #############################
 # The metric reporter to collect and report metrics for Auto Balancer
 metric.reporters=kafka.autobalancer.metricsreporter.AutoBalancerMetricsReporter
 
-# The network inbound bandwidth capacity, used in NetworkInCapacityGoal and calculation of inbound bandwidth utilization, default 100MB/s
-autobalancer.reporter.network.in.capacity=102400
+# The network inbound bandwidth in bytes/s, default 100MB/s. Used in NetworkInCapacityGoal and calculation of inbound bandwidth utilization
+autobalancer.reporter.network.in.capacity=104857600
 
-# The network outbound bandwidth capacity, used in NetworkOutCapacityGoal and calculation of outbound bandwidth utilization, default 100MB/s
-autobalancer.reporter.network.out.capacity=102400
+# The network outbound bandwidth in bytes/s, default 100MB/s. Used in NetworkOutCapacityGoal and calculation of outbound bandwidth utilization
+autobalancer.reporter.network.out.capacity=104857600
```

Comment:
These configs are also set in `tests/kafkatest/services/kafka/templates/kafka.properties`, which should be updated as well.